cypress/integration/add-delete-collection.ts

```diff
@@ -0,0 +1,36 @@
+before(() => {
+  cy.clearCookies()
+  cy.clearLocalStorage()
+})
+
+it('can add cards to the collection', () => {
+  cy.visit('/')
+  cy.findAllByRole('link', { name: 'Search' }).click()
+  const cardFigures = cy.findAllByRole('figure').should('exist')
```

Comment:
you can't use variables in Cypress like this: https://docs.cypress.io/guides/core-concepts/variables-and-aliases

<img width="694" alt="Screen Shot 2021-05-21 at 2 40 43 PM" src="https://user-images.githubusercontent.com/37197876/119195867-8b250880-ba42-11eb-94e4-a65a69fb4ede.png">